src/screens/Contact/Contact.tsx

```diff
@@ -1,145 +1,31 @@
-import React, { FunctionComponent } from 'react';
+import React, { Fragment } from 'react';
 import AppMetaTags from 'components/AppMetaTags/AppMetaTags';
-import ShareBlock from 'components/ShareBlock/ShareBlock';
-import Typography from '@material-ui/core/Typography';
-import StapledSidebar, {
-  SidebarLink,
-  SectionHeader,
-} from 'components/StapledSidebar/StapledSidebar';
-import { Wrapper, Content, Header } from './Contact.style';
-import ExternalLink from 'components/ExternalLink';
-import { Link } from 'react-router-dom';
-
-const sidebar = (
-  <React.Fragment>
-    <SidebarLink href="#general">General Questions</SidebarLink>
-    <SidebarLink href="#governments">Governments</SidebarLink>
-    <SidebarLink href="#press">Press</SidebarLink>
-    <SidebarLink href="#join-us">Join Us</SidebarLink>
-    <SidebarLink href="#donate">Donate</SidebarLink>
-  </React.Fragment>
-);
-
-const Paragraph: FunctionComponent = ({ children }) => (
-  <Typography variant="body1" component="p">
-    {children}
-  </Typography>
-);
-
-const ListItem: FunctionComponent = ({ children }) => (
-  <li>
-    <Paragraph>{children}</Paragraph>
-  </li>
+import { Heading1, Heading2, MarkdownContent } from 'components/Markdown';
+import { Anchor } from 'components/TableOfContents';
+import MainContent from 'components/PageContent';
```

Comment:
Thoughts on trying to keep naming consistent and calling this `PageContent` instead of `MainContent` (or renaming the component/file as `MainContent`)? Sometimes the inconsistent naming makes things read a bit confusing

Reply:
agree, I think `PageContent` is better 👍